src/allocator.c

```diff
@@ -542,15 +542,15 @@ _dispatch_alloc_maybe_madvise_page(dispatch_continuation_t c)
 	}
 	// They are all unallocated, so we could madvise the page. Try to
 	// take ownership of them all.
-	int last_locked = 0;
-	do {
+	unsigned int last_locked;
```

Comment:
This change here was already turned down as part of PR https://github.com/apple/swift-corelibs-libdispatch/pull/564 so it really shouldn't be reintroduced here.

Reply:
Well for starters, it's the only odd loop out of the others that enumerate the bitmap, and the only one that casts bitmaps to page to signed for no reason. Or at least, whatever reason it was in the past, isn't relevant now. It's making a compromise for nothing. It should be a simple, canonical, similar in style, for loop.